packages/builder/src/link.js

```diff
@@ -10,6 +10,11 @@ import getSkpmConfigFromPackageJSON from '@skpm/internal-utils/skpm-config'
 import getSketchVersion from '@skpm/internal-utils/getSketchVersion'
 import checkDevMode from '@skpm/internal-utils/check-dev-mode'
 
+// Linking makes no sense for `npm ci`, so let's do nothing here
```

Comment:
I think it's the right fix but the comment isn't correct: `npm ci` has nothing to do with `process.env.CI`. The env variable is set by pretty much all CI providers, not by npm.

```suggestion
// Linking makes no sense when running on a CI, so let's do nothing here
```

Reply:
🙌